### PR TITLE
howtos的Overview校对

### PR DESCRIPTION
**Pull request by andyyehoo**

### CODE REVIEW

**Comment by lucky521** [comment]
Thank you! 

Tips: 
To avoid duplicated work, you should create PR to README.md to claim your work before you start to translate or proofread.

https://github.com/jikexueyuanwiki/tensorflow-zh/pull/89
Another friend proofread `how_tos/overview.md`, too.

**Comment by dywlegend1002** [comment]
@dywlegend1002

File: SOURCE/how_tos/overview.md [hunk 1]
```diff
@@ -1,78 +1,78 @@
...
-当在多 GPU 上部署大型的模型，或展开复杂的 LSTMs 或 RNNs 时，在模型构建代码的不同位置对许多相同的 Variable 对象(objects)进行读写常常是必须的。
+当在多GPU上部署大型的模型，或展开复杂的LSTMs或RNNs，经常需要在模型构建代码的不同位置，对相同的变量对象进行访问
 
-设计"Variable Scope" 机制的目的就是为了有助于上述任务的实现。
+"变量范围"机制设计的目的就是实现这个
 
-[参看教程](../how_tos/variable_scope/index.md)。
+[参看教程](../how_tos/variable_scope/index.md)
 
-原文： [How-to](http://tensorflow.org/how_tos/index.html) 翻译：[Terence Cooper](https://github.com/TerenceCooper) 校对：
```
Comment (dywlegend1002) [inline SOURCE/how_tos/overview.md]: dywlegend1002

File: README.md [hunk 2]
```diff
@@ -91,8 +91,8 @@ PS: 想探讨TensorFlow技术的可以加"TensorFlow技术交流群"：495115006
   - [曼德布洛特(Mandelbrot)集合](SOURCE/tutorials/mandelbrot.md) 翻译：([@ericxk](https://github.com/ericxk))√  校对： ([@tensorfly](https://github.com/tensorfly))√
   - [偏微分方程](SOURCE/tutorials/pdes.md) 翻译：([@wangaicc](https://github.com/wangaicc))√  校对： ([@tensorfly](https://github.com/tensorfly))√
```
Comment (dywlegend1002) [inline README.md]: @dywlegend

**Pull request closed without merging**